services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -63,13 +66,23 @@ export const useOverviewLoaded = (): boolean => useOverview(({ loaded }) => load
 
 const emptyBatch: BatchRequest = { actions: [] };
 export const [useAction, UpdateAction] = createStore(emptyBatch);
-
+const tagsResponse: GetGitTagsResponse = { tagData: [] };
+export const refreshTags = (): void => {
+    const api = useApi;
+    api.tagsService()
+        .GetGitTags({})
+        .then((result: GetGitTagsResponse) => {
+            updateTag.set(result);
+        });
```

Comment:
we need an error handler here.
```suggestion
        }).catch(...;
```
It should call `showSnackbarError` to display the error.